### PR TITLE
this is wrong

### PR DESCRIPTION
**Pull request by joshyoon2020**

### CODE REVIEW

**Comment by joshyoon2020** [comment]
@twang

File: unit1workinprogress.rtf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        squares();\
+        System.out.println("+------+ +------+");\
+        triangles();\
+    \}\
+    public static void triangles()\{\
+        System.out.println("   /\\\\       /\\\\");\
+        System.out.println("  /  \\\\     /  \\\\");\
+        System.out.println(" /    \\\\   /    \\\\");\
+    \}\
+    public static void squares()\{\
```
Comment (asarathy17) [inline unit1workinprogress.rtf]: You have two methods called square!!!! You need to change them.

File: unit1workinprogress.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        System.out.println(" /    \\   /    \\");
+    }
+    public static void squares(){
+        System.out.println("+------+ +------+");
+        System.out.println("|      | |      |");
+        System.out.println("|      | |      |");
+       
+            
+    }
+    public static void squares(){
```
Comment (asarathy17) [inline unit1workinprogress.java]: This is called squares. Your other method is also called squares.

**Comment by twang** [comment]
Thanks @joshyoon2020 and @asarathy17!

**Pull request closed without merging**